packages/kotti-ui/source/kotti-field-text-area/KtFieldTextArea.vue

```diff
@@ -27,9 +31,26 @@ export default defineComponent({
 			props,
 			supports: KOTTI_FIELD_TEXT_AREA_SUPPORTS,
 		})
+		const textareaRef = ref<HTMLTextAreaElement | null>(null)
 
 		const { forceUpdate, forceUpdateKey } = useForceUpdate()
 
+		const resizeTextarea = (
+			textarea: HTMLTextAreaElement | null,
+			autoSize: boolean,
+			maxHeightInPixels: number | null,
+		) => {
+			if (
+				autoSize &&
+				textarea &&
+				((maxHeightInPixels && textarea?.scrollHeight < maxHeightInPixels) ||
```

Comment:
Is it just `<` or `<=`?

the wording `max height` implies that it is inclusive, no? so it should be `<=`